changelogs/fragments/5097-terraform-generate-tfvars.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - terraform - adds ``generate_variables`` parameter for automatically declaring the Terraform vars from Ansible (https://github.com/ansible-collections/community.general/pull/5097).
```

Comment:
Is it `generate_variables` or `declare_variables`? Please choose one name and use it consistently, everywhere.

Reply:
This still needs to be addressed.